docs/changes/1.0.0.rst

```diff
@@ -94,7 +94,7 @@ New & Improved
     api.add_route('/files/{file_id}', resource_1)
     api.add_route('/files/{file_id}.{ext}', resource_2)
 
-- Support was added to :any:`API.add_route()` for passing through
```

Comment:
Could you make sure all the changelogs for Falcon ⩽ 2.0.0 are not tampered with, as requested by @nZac ?

Reply:
Whoops, I resolved this accidentally 😬. Sorry, this still needs to get cleaned up

Reply:
OK, is this the last thing we are waiting on?

Reply:
yea, just the changelog stuff. I didn't get to it yesterday and I wont get to it this week.

Reply:
@kgriffs @vytas7 I'll happy if someone able to fix that last thing.

This week I have a limited amount of time, also need to finish this guy https://github.com/falconry/falcon/pull/1557